assemblies/pentaho-cde-renderer/feature/src/main/feature/feature.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<features name="${project.artifactId}-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.0.0">
+
+  <feature name="foundry-cde-renderer" version="${project.version}">
+    <feature version="${pentaho.version}">pentaho-requirejs-osgi-manager</feature>
+    <feature version="${pentaho.version}">pentaho-webpackage</feature>
+  </feature>
+
+  <feature name="pentaho-requirejs-osgi-manager" version="${pentaho.version}">
+    <feature>http</feature>
+
+    <bundle>mvn:commons-lang/commons-lang/2.6</bundle>
+    <bundle>wrap:mvn:com.github.zafarkhaja/java-semver/0.9.0</bundle>
+    <bundle>mvn:pentaho/pentaho-requirejs-utils/${pentaho.version}</bundle>
+    <bundle>mvn:pentaho/pentaho-requirejs-osgi-manager/${pentaho.version}</bundle>
```

Comment:
Use the proper version property name: https://github.com/pentaho/build-resources/blob/8.1.0.0/version.properties#L101